.travis.yml

```diff
@@ -0,0 +1,25 @@
+---
+language: node_js
+node_js:
+  - "12"
+
+dist: xenial
+
+addons:
+  chrome: stable
+
+cache:
+  directories:
+    - $HOME/.npm
+
+env:
+  global:
+    # See https://git.io/vdao3 for details.
+    - JOBS=1
+
+branches:
+  only:
+    - master
+
+script:
+  - npm test
```

Comment:
This project uses GitHub Actions for continuous integration. I forgot to mention, after letting Ember CLI run, you will want to delete `.travis.yml`.

Reply:
ok I will delete it.